manifests/server/service.pp

```diff
@@ -18,7 +18,7 @@
     fail("Use of private class ${name} by ${caller_module_name}")
   }
 
-  if $sensu::manage_services {
+  if $sensu::manage_services and !$sensu::enterprise {
```

Comment:
Same thing as above with the api service - can we merge the conditionals together so we aren't defining the service multiple times?